components/popover_list_members/popover_list_members.tsx

```diff
@@ -80,7 +91,7 @@ export default class PopoverListMembers extends React.PureComponent {
         this.setState({showPopover: false});
     };
 
-    showMembersModal = (e) => {
+    showMembersModal = (e: { preventDefault: () => void }) => {
```

Comment:
this can be typed with one of Reacts built in Event-types

Reply:
Done